content/automate/ManageIQ/System/CommonMethods/Utils.class/__methods__/log_object.rb

```diff
@@ -42,6 +42,48 @@ def self.log(obj, object_string = 'Automation Object', handle = $evm)
               obj.attributes.sort.each { |k, v| handle.log("info", "   Attribute - #{k}: #{v}") }
               handle.log("info", "Listing #{object_string} Attributes - End")
             end
+
+            def self.log_ar_objects(object_string = 'VMDB Object', handle = $evm)
```

Comment:
@billfitzgerald0120 I think instead of using object_string we should call it log_prefix.